### PR TITLE
Fixed a variety of vanilla model and hitbox errors

### PR DESCRIPTION
**Pull request by HasteDC**
This fixes the following issues:

- The top fin of bh_elite has a small hole at the back. This has been closed. After noticing this same hole was already fixed in bh_vheavy_fighter, I slightly altered the UV mapping of the fixed hole there so it matches bh_elite.
- The cockpit of rh_fighter has broken texturing on the starboard side. I fixed it to match the port side completely.
- The cockpit glass of ku_fighter has a hole on it on both sides. I closed that hole.
- The Corsair shipline (which, of course, is still in \PIRATE\) has several problems in all of their hitboxes. I corrected their shield hitboxes (HpMount in the .sur file) to be "flipped" the right way, and to be symmetrical. Their face count and size has not been affected. For the Titan (pi_vheavy_fighter) I slightly adjusted the overall shape so it properly covers the whole ship while minimally impacting overall size. In addition, all mismatched (usually rotated 90 degrees in a few directions) wing and fin hitboxes have been fixed for all three of the fighters.

### CODE REVIEW

**Comment by erikroe** [comment]
@thaddeus-ex This PR would conflict with your Drake since you remade the wireframes. On the other models, the FLSR-specific Hardpoints are missing.

**Comment by HasteDC** [comment]
I've made no hardpoint changes in any of these, and the .cmps used were the latest available on the repository. I am of course not super familiar with the mod, but at a glance it looks like FLSR-specific hardpoints like HpModule01 are present in all of the .cmps I modified.

The .surs for the Corsair ship family were pulled from vanilla and then updated, as none were previously present (so FLSR was presumably working with the vanilla hitboxes here).

Swapping out the wireframe of the Drake is a minute's worth of work if a new one has been made that isn't yet present on the repository.

**Comment by erikroe** [comment]
Heyo, there's a few things about this PR:
- BH VHF hole was already fixed in FLSR
- Cosair line of ships may have the shield bubble rotated correctly, but it does not wrap properly around all the guns. Thaddeus was working on this already anyway and almost finished this.
- the normals of the gap in the glass of the Drake’s cockpit are wrong – Thaddeus is working on this ship in general currently, too
- bh_elite VMS may need the proper path in its name to avoid any possible conflicts

Basically from the PR we would take the bh_elite if fixed, the rest is and was already in the works or already done and maybe just not yet properly included.

**Comment by erikroe** [comment]
Thaddeus is going over every ship by now. Most if not all of the ones here have been touched by him already, with said improvements included.
